main.go

```diff
@@ -164,6 +176,11 @@ func NewApp() *cli.App {
 			Name:  "status",
 			Usage: "List applied and pending migrations",
 			Flags: []cli.Flag{
+				&cli.BoolFlag{
```

Comment:
There is a precedent for duplication with the verbose flag, but it seems like it would be better to define the flag just once

```
strictFlag := cli.BoolFlag{...}
flags : []cli.Flag{&strictFlag}
```

Reply:
That could really tidy up the duplicate flag usage. I was already a little annoyed writing that part but I tried to do it similar to the verbose flag. I will change it 👍